module/documents/actor/actor.mjs

```diff
@@ -753,6 +753,20 @@ export default class Actor5e extends Actor {
     return allowed !== false ? this.update(updates, {dhp: -amount}) : this;
   }
 
+  /**
+   * Apply a certain amount of tempoary hit point, but only if it's more then the actor currently has.
+   * @param {number} amount       An amount of temporary hit points to set
+   * @returns {Promise<Actor5e>}  A Promise which resolves once the temp HP has been applied
+   */
+  async applyTempHp(amount = 0) {
```

Comment:
I think the HP in the method name should be capitalized to match methods like `_prepareNPCData`.

```suggestion
  async applyTempHP(amount=0) {
```